cypress/integration/identifier/PersistentLinks.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Persistent Links
+  As a user
+  I want to click on a link with an 'href' having an unmodifiable id of a user
```

Comment:
I doubt this is how a user of the site would think about it, but maybe I am just being pedantic.... 
I would expect them to think more along the lines of `I want to avoid clicking on links that lead nowhere or an error page`